frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -520,6 +520,7 @@ const DeviceUserPage = ({
             hostDisplayName={host.display_name}
             software={selectedSoftwareDetails}
             onExit={() => setSelectedSoftwareDetails(null)}
+            isMyDevicePage
```

Comment:
i think this should be called something like `showDetailsContent`. That way the `SoftwareDetailsModal` is not concerned who its parent is.

Reply:
Code looks good, what @ghernandez345 makes sense for re usability and better semantic clarity to describe the singular purpose.